.config/nextest.toml

```diff
@@ -1,8 +1,9 @@
 [profile.default]
-retries = 2
+retries = { backoff = "fixed", count = 3, delay = "3s" }
 
 [profile.ci]
+retries = { backoff = "fixed", count = 3, delay = "3s" }
```

Comment:
No need to set it twice - `retries` from `default` profile will be picked up anyway (see [hierarchical configuration](https://nexte.st/docs/configuration/#hierarchical-configuration)).